### ISSUE TITLE
Regular comment "//" inside comment block "/* */" does not remove highlighting.

### ISSUE THREAD

**Issue by Glinkis** [CoenraadS/BracketPair#88]
```
/*
 * ([{}]) <= not highlighted
 * // <= this is what actives the highlighting
 * ([{}]) <= highlighted
 */
```

**Comment by CoenraadS** [comment]
Tracking here: https://github.com/PrismJS/prism/issues/1340

**Comment by CoenraadS** [comment]
Fixed in v1.0.20
